manager/integration/tests/test_basic.py

```diff
@@ -519,6 +502,115 @@ def backupstore_test(client, host_id, volname, size):
     assert not found
 
 
+@pytest.mark.coretest   # NOQA
+def test_listing_backup_volume(clients, base_image=""):   # NOQA
+    for host_id, client in clients.iteritems():
+        break
+    lht_hostId = get_self_host_id()
+
+    # create 2 volumes.
+    volume1_name = "list-backup-volume-test-1"
+    volume2_name = "list-backup-volume-test-2"
+    volume1 = create_volume(client, volume1_name)
+    volume2 = create_volume(client, volume2_name)
+    volume1.attach(hostId=lht_hostId)
+    volume1 = common.wait_for_volume_healthy(client, volume1_name)
+    volume2.attach(hostId=lht_hostId)
+    volume2 = common.wait_for_volume_healthy(client, volume2_name)
+
+    setting = client.by_id_setting(common.SETTING_BACKUP_TARGET)
+    backupstores = common.get_backupstore_url()
+    for backupstore in backupstores:
+        if common.is_backupTarget_nfs(backupstore):
+            nfs_url = backupstore.strip("nfs://")
+            setting = client.update(setting, value=backupstore)
+            assert setting["value"] == backupstore
+
```

Comment:
Assert backupstore is `NFS` based. (you may need to retry and wait for the entry to update)